DeviceTenantFinder/Src/DeviceTenantFinder/Program.cs

```diff
@@ -85,7 +85,7 @@ public static async Task<int> Main(string[] args)
                 Devices devices = JsonConvert.DeserializeObject<Devices>(result);
                 foreach (Item item in devices.Items)
                 {
-                    if (item.DeviceId == args[0])
```

Comment:
Just to note that `string.Equals` can take a `StringComparison` parameter which allows you to specify case-insensitive comparisons: https://docs.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings